crates/bevy_render/src/camera/camera.rs

```diff
@@ -886,6 +893,23 @@ pub fn extract_cameras(
 
             let mut commands = commands.get_or_spawn(entity);
 
+            let target_format = if target_formats.is_supported(camera.target_format) {
```

Comment:
Does it really make sense to fallback here rather than just panic? Is the intention here to be defensive for developers on mobile who might accidentally choose the wrong format for a device? This seems like a dev time bug to me rather than something we'd want to gracefully fail but I'm not sure.